webextensions/api/browserAction.json

```diff
@@ -595,7 +595,7 @@
                 },
                 "edge": "mirror",
                 "firefox": {
-                  "version_added": true,
+                  "version_added": "≤57",
```

Comment:
@queengooborg as the PR that added this is dated Jan 19, 2018, shouldn't this be version 63?